src/cr/cube/matrix/assembler.py

```diff
@@ -85,16 +85,16 @@ def column_weighted_bases(self):
     def columns_base(self):
         """1D/2D np.int64 ndarray of unweighted-N for each slice column/cell."""
         # --- an MR_X slice produces a 2D column-base (each cell has its own N) ---
-        if self._rows_dimension.dimension_type == DT.MR_SUBVAR:
+        columns_base = self._cube_result_matrix.columns_base
+        rows_dim_type = self._rows_dimension.dimension_type
+        if rows_dim_type in (DT.MR_SUBVAR, DT.NUM_ARRAY) and columns_base.ndim > 1:
```

Comment:
Is there a case when the first part of the `and` is satisfied, but the second is not? I'm curious to see if we have an array where the columns base `ndim` would be 1.

Reply:
yes num_array no group should have the first condition true and the second false